src/PyFOOOF.jl

```diff
@@ -0,0 +1,52 @@
+module PyFOOOF
+
+using PyCall
+
+#####
+##### Exports
+#####
+
+#####
+##### init
+#####
+
+const fooof = PyNULL()
+
+function __init__()
+    # all of this is __init__() so that it plays nice with precompilation
+    # see https://github.com/JuliaPy/PyCall.jl/#using-pycall-from-julia-modules
+    copy!(fooof, pyimport("fooof"))
+    # don't eval into the module while precompiling; this breaks precompilation
+    # of downstream modules
+    if ccall(:jl_generating_output, Cint, ()) == 0
+        # delegate everything else to fooof
+        for pn in propertynames(fooof)
+            isdefined(@__MODULE__, pn) && continue
+            prop = getproperty(fooof, pn)
+            @eval $pn = $prop
+        end
+    end
+    return nothing
+end
+
+"""
+    install_matplotlib(ver="")
```

Comment:
I feel like installing matplotlib should just be the user's responsibility. That is, it seems odd to me for a package meant to wrap FOOOF to offer to install a different package. We could direct users to PyPlot if they want to interface with FOOOF's matplotlib functionality.